apispec/psc-statement-delta-spec.yml

```diff
@@ -48,6 +48,8 @@ components:
           type: string
         linked_psc:
           $ref: '#/components/schemas/LinkedPsc'
+        restrictions_notice_withdrawal_reason:
```

Comment:
I think CHIPS side this needs to be restrictions_notice_reason. See https://github.com/companieshouse/private-api-sdk-java/blob/master/generated_sources/src/main/java/uk/gov/companieshouse/api/delta/PscStatement.java#L36

Reply:
https://github.com/companieshouse/chs-transform-api/blob/master/lib/ChsTransformApi/Transforms/Mapping/PSCStatement/Statement.pm#L32 for the original perl